### ISSUE TITLE
How do I use IssueNotes.all & IssueDiscussion.all

### ISSUE THREAD

**Issue by myayop** [jdalrymple/gitbeaker#952]
Tried the below code snippet : 

await api.IssueNotes.all({projectId: 170, issueId: 342}).then(notes => {
      console.log(notes);
    });

got error response:

HTTPError: Response code 400 (Bad Request)
    at onResponse (/Users/sheikhnaziruddin/gitlab/gitbeaker/node_modules/got/dist/source/as-promise/index.js:132:28)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  name: 'HTTPError',
  code: undefined,
  timings: {
    start: 1594221381860,
    socket: 1594221381860,
    lookup: 1594221381861,
    connect: 1594221381877,
    secureConnect: 1594221381902,
    upload: 1594221381902,
    response: 1594221381956,
    end: 1594221381959,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 0,
      dns: 1,
      tcp: 16,
      tls: 25,
      request: 0,
      firstByte: 54,
      download: 3,
      total: 99
    }
  },
  description: undefined
}

Look closely at ResourceNotes.ts, found that

line 26 : return RequestHelper.get(this, `${rId}/${this.resource2Type}/${r2Id}/notes`, options);

shouldn't it be 'projects/${rId}/${this.resource2Type}/${r2Id}/notes' instead of '${rId}/${this.resource2Type}/${r2Id}/notes'

Tested against Gitlab version :  12.9.0

**Comment by jdalrymple** [comment]
This url change happens in the background, Ill give it a look.

**Comment by rok-star** [comment]
So guys, what news? Experiencing the same issue!

**Comment by jdalrymple** [comment]
The arguments are not supposed to be in a form of an object, since they are not optional. So instead of:
```
await api.IssueNotes.all({projectId: 170, issueId: 342})...
```

It should be:
```
await api.IssueNotes.all(170, 342)...
```

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`23.3.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/23.3.0) :rocket:
